### PR TITLE
Feature/genai poc

### PR DESCRIPTION
**Pull request by martingaleano**
## What?
* Create VPC in data-science account
* Create ECS cluster together with a service for b2chat
* Create ACM certificate
* Create ALB
* Create route53 record in public hosted zone 

## Why?
* https://github.com/binbashar/le-tf-infra-aws/issues/611

## References
* Link to any supporting github issues or helpful documentation to add some context (e.g. stackoverflow).
* Use `closes #123`, if this PR closes a GitHub issue `#123`

### CODE REVIEW

**Review by exequielrafaela — APPROVED** [review]

**Review by exequielrafaela — CHANGES_REQUESTED** [review]
@martingaleano Tincho let's please use binbash.co instead of binbash.com.ar domain. Thanks!

**Comment by martingaleano** [comment]
> @martingaleano Tincho let's please use binbash.co instead of binbash.com.ar domain. Thanks!

Thanks @exequielrafaela . I've just moved the domain to binbash.co

**Review by exequielrafaela — APPROVED** [review]

File: data-science/us-east-1/genai-llm-rag-bedrock-poc/locals.tf [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    Environment = var.environment
+  }
+
+  container_definitions = {
+    b2chat = {
+      image                     = "905418344519.dkr.ecr.us-east-1.amazonaws.com/bb-data-science-genai-llm-rag-bedrock-poc-b2chat:latest"
+      enable_cloudwatch_logging = false
+      readonlyRootFilesystem    = false
+      cpu = 1024
+      memory = 2048
```
Comment (diego-ojeda-binbash) [inline data-science/us-east-1/genai-llm-rag-bedrock-poc/locals.tf:15]: @martingaleano does this container require this much resources?

Comment (martingaleano) [reply data-science/us-east-1/genai-llm-rag-bedrock-poc/locals.tf]: @diego-ojeda-binbash  No, it doesn't. I've just updated the PR and replaced those values with cpu = 512 and memory = 1024. I think  these values fit much better.

**Review by diego-ojeda-binbash — APPROVED** [review]

**Pull request merged**